.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+yarn-error.log
```

Comment:
Can we just change lines 2 and 3 to `*.log`?  Why didn't we get the express generator .gitignore?